webapp/app/forms/steps/eligibility_steps.py

```diff
@@ -62,16 +118,19 @@ class DecisionEligibilityInputFormSteuerlotseStep(EligibilityInputFormSteuerlots
 
     def __init__(self, *args, **kwargs):
         super(DecisionEligibilityInputFormSteuerlotseStep, self).__init__(*args,
-            **kwargs,
-            )
+                                                                          **kwargs,
+                                                                          )
```

Comment:
I think I do not really get your comment. Do you just want one line?

BTW: It would help me a lot if you could (especially for something like this) include a few more lines around the important part for context 😬

Reply:
Yes, just one line, exactly.

It's weird, but I feel like GitHub did that for me for a while there, they just seem to have stopped. But maybe I'm wrong. I'll try my best. :)